RoadMap_UIKit/RoadMap_UIKit/ReaderViewController.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  ViewController.swift
+//  RoadMap_UIKit
+//
+//  Created by Артур Щукин on 21.09.2022.
+
+import UIKit
+
+// Стартовый экран приложения
+final class ReaderViewController: UIViewController {
+    
+    // MARK: - Constants
+    enum Constants {
+        static let fonts = ["Palatino", "Optima Regular", "Kohinoor Devanagari Regular", "Georgia"]
+    }
+    // MARK: - IBOutlets
+    // MARK: - Visual Components
+    private let redTextButton = UIButton()
+    private let systemPurpleButton = UIButton()
+    private let systemBrownButton = UIButton()
+    private let systemTealButton = UIButton()
+    private let regularFontButton = UIButton()
+    private let boldFontButton = UIButton()
+    
+    private let textFontSizeSlider = UISlider()
+    
+    private let fontPicker = UIPickerView()
+    
+    private let darkModeSwitch = UISwitch()
+    
+    // MARK: - Public properties
+    let fontTextField = UITextField()
+    
+    var myTextView = UITextView()
+    // MARK: - Private properties
```

Comment:
```suggestion
    // MARK: - Private properties
```
убери марки которые не используешь